app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -77,21 +75,6 @@
 public class DocumentEditFragment extends GsFragmentBase implements TextFormat.TextFormatApplier {
     public static final String FRAGMENT_TAG = "DocumentEditFragment";
     public static final String SAVESTATE_DOCUMENT = "DOCUMENT";
-    private static final String SAVESTATE_CURSOR_POS = "CURSOR_POS";
-    private static final String SAVESTATE_PREVIEW_ON = "SAVESTATE_PREVIEW_ON";
```

Comment:
We don't need to use the instance state, we are already tracking these with `_appSettings`. Removing redundant information

Reply:
What I would pursue in future is similar data class to dopt with props cursorPos, previewOn...etc. With read/write from json file. Instead of SharedPreferencesBackend i.e. HashMap backend with r/w from that json file.

But please even if you like it - don't make changes for that idea in this pull request, otherwise it's just growing :smile: